packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -56,8 +56,12 @@ export interface IPushOptions {
     channelId?: string;
     categoryId?: string;
     mutableContent?: boolean;
-    android?: { [key: string]: { [key: string]: string } };
-    apns?: { payload: { aps: { [key: string]: { [key: string]: string } } } };
+    fcmOptions?: { [key: string]: string };
+    android?: object;
+    apns?: {
+      headers?: { [key: string]: string };
```

Comment:
Update the PR to provide just this change. And please also update the tests or add a new one to check this new property is passed properly.